### ISSUE TITLE
Visualisation errors and computing efficiencies

### ISSUE THREAD

**Issue by Frederik-Roeckle** [Frederik-Roeckle/provibackend#59]
**TLDR** 

2 visual errors risen up with the new adjustments, compute time is reaching high limits, need for reduction in compute time. This is related to the the two Merge Requests from you @luisabuck.

---

- **Issues with the normal model**

The path occurence frequency is not properly placed and needs adjustment

![Image](https://github.com/user-attachments/assets/d6bcb28a-82f4-4d4d-8cc3-2e66e246b331)

- **Issues in the Mental Map Model:**

Issue: There is a grayish box on the top right corner that needs to be removed
![Image](https://github.com/user-attachments/assets/1e1ad590-0b2b-41a5-b8e3-c3e8992b938a)

- **Compute Efficiency**
Overall it would be very important to reduce the compute time for the graphs. One of the biggest cost attributer is the  `xes_importer.apply(base_xes)` function. Therefore reducing the occurence of this statement is highly appreciated. Please try to reduce it to a minima.

**Comment by luisabuck** [comment]
Grayish Box is removed; Solutions w.r.t the label placements sent to Marie for decision

**Comment by luisabuck** [comment]
Edge Label Frequencies are removed
